### PR TITLE
docs(readme): document 'pipx' alternative installation steps

### PR DESCRIPTION
**Pull request by AdrianDC**
## Description

**docs(readme): document 'pipx' alternative installation steps**

Closes: #1113

## Checklist

- [ ] Add test cases to all the changes you introduce
- [ ] Run `./scripts/format` and `./scripts/test` locally to ensure this change passes linter check and test
- [ ] Test the changes on the local machine manually
- [x] Update the documentation for the changes

## Expected behavior

-

## Steps to Test This Pull Request

-

## Additional context

-

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -46,10 +46,18 @@ descriptive commits.
 
 ## Installation
 
-To make commitizen available in your system
+To install commitizen in your system, if you are using `pipx` (Recommended, <https://pypa.github.io/pipx/installation/>):
```
Comment (Lee-W) [inline docs/README.md:49]:
```suggestion
Install commitizen in your system using `pipx` (Recommended, <https://pypa.github.io/pipx/installation/>):
```

File: docs/README.md [hunk 2]
```diff
@@ -46,10 +46,18 @@ descriptive commits.
...
 
 ```bash
-pip install --user -U Commitizen
+pipx ensurepath
+pipx install commitizen
+pipx upgrade commitizen
+```
+
+To install commitizen in your system, if you are using `pip`:
```
Comment (Lee-W) [inline docs/README.md:57]:
```suggestion
Install commitizen using `pip` with `--user` flag:
```

**Review by Lee-W — COMMENTED** [review]
Overall, looks great! add 2 minor rewording comments

File: docs/README.md (hunk 1 above)
Comment (AdrianDC) [reply docs/README.md]: Thanks, better :+1:

File: docs/README.md (hunk 2 above)
Comment (AdrianDC) [reply docs/README.md]: Thanks, better :+1:

**Review by Lee-W — APPROVED** [review]

**Pull request merged**